pages/common/stolonctl.md

```diff
@@ -0,0 +1,19 @@
+# Stolon
+
+> Stolon is a cloud native PostgreSQL manager for PostgreSQL high availability.
+
+- Get cluster status:
+
+`stolonctl --cluster-name ${{stolon_cluster_name}} --store-backend ${{stolon_store_backend}} --store-endpoints ${{stolon_store_endpoints}} status`
```

Comment:
I think we can remove "stolon_" from the token values. It is clear enough that the items are about stolon.